pf4j/src/main/java/ro/fortsoft/pf4j/PluginDependency.java

```diff
@@ -12,38 +12,38 @@
  */
 package ro.fortsoft.pf4j;
 
+import com.github.zafarkhaja.semver.Version;
+
 /**
  * @author Decebal Suiu
  */
 public class PluginDependency {
 
-	private String pluginId;
-	private Version pluginVersion;
-
-	public PluginDependency(String dependency) {
-		/*
-		 int index = dependency.indexOf(':');
-		 if (index == -1) {
-			 throw new IllegalArgumentException("Illegal dependency specifier "+ dependency);
-		 }
-
-		 this.pluginId = dependency.substring(0, index);
-		 this.pluginVersion = Version.createVersion(dependency.substring(index + 1));
-		 */
-		this.pluginId = dependency;
-	}
-
-	public String getPluginId() {
-		return pluginId;
-	}
-
-	public Version getPluginVersion() {
-		return pluginVersion;
-	}
-
-	@Override
-	public String toString() {
-		return "PluginDependency [pluginId=" + pluginId + ", pluginVersion=" + pluginVersion + "]";
-	}
+    private String pluginId;
+    private String pluginVersionSupport;
+
+    public PluginDependency(String dependency) {
+        int index = dependency.indexOf('@');
```

Comment:
For me the sintax plugin-at-version makes more sense but I saw that you were using : and forgot to change it.